docs/architecture.md

```diff
@@ -118,7 +118,7 @@ after the VirtualMachine was created, but before it started:
 All service exposure options that apply to a VirtualMachineInstance
 apply to a VirtualMachine.
 
-See [Network Service Integration](usage/network-service-integration.md) for more details.
+See [Network Service Integration](virtual_machines/service_objects.md) for more details.
```

Comment:
This is incorrect.  Please remove this diff.

See https://github.com/kubevirt/user-guide/pull/389/commits/8e55dc19a7a86cbc073ac303d4ee94ede3a3bb39

Reply:
OK I've removed this completely, but I was wondering, would `[Network Service Integration](./virtual_machines/service_objects)` work?

- [x] Remove service object link change